Week3/homework/3-step3.js

```diff
@@ -2,24 +2,31 @@
 
 // use a 'for' loop
 function repeatStringNumTimesWithFor(str, num) {
-  // add your code here
-  return str;
+  let x = '';
+  for (let i = 0; i < num; i++) {
+    x = x + str;
```

Comment:
you have a :1st_place_medal: for this :point_up:, But can you make it work without `x`?

Same the next two pieces.